packages/rum-core/src/domain/waitIdlePageActivity.ts

```diff
@@ -154,16 +59,84 @@ export function waitPageActivitiesCompletion(
     clearTimeout(validationTimeoutId)
     clearTimeout(idleTimeoutId)
     clearTimeout(maxDurationTimeoutId)
-    stopPageActivitiesTracking()
+    pageActivitySubscription.unsubscribe()
   }
 
-  function complete(params: CompletionCallbackParameters) {
+  function complete(event: IdlePageActivityEvent) {
     if (hasCompleted) {
       return
     }
     stop()
-    completionCallback(params)
+    idlePageActivityCallback(event)
   }
-
   return { stop }
 }
+
+// Automatic action collection lifecycle overview:
+//                      (Start new trackPageActivities)
```

Comment:
`trackPageActivities` does not exist anymore. Also, it could make sense to move this graph higher in the file.

Reply:
Agreed